zebra-consensus/src/config.rs

```diff
@@ -4,14 +4,14 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
-    /// Should Zebra sync using checkpoints?
+    /// Should Zebra sync using post-Canopy checkpoints?
     ///
-    /// Setting this option to true enables post-Canopy checkpoints.
-    /// (Zebra always checkpoints up to and including Canopy activation.)
+    /// This option is `true` by default, and allows for faster chain synchronization.
     ///
-    /// Future versions of Zebra may change the mandatory checkpoint
-    /// height.
+    /// Disabling this option forces Zebra to only use checkpoints until Canopy activation, which
+    /// helps with debugging by forcing Zebra to validate more blocks.
```

Comment:
This is all still true, we plan on changing the required and optional checkpoints regularly.

```suggestion
    /// Zebra requires some checkpoints to validate legacy network upgrades.
    /// But it also ships with some optional checkpoints,
    /// which are used instead of full validation by default.
    ///
    /// Disabling this option makes Zebra start full validation as soon as possible.
    /// This helps developers debug Zebra, by running full validation on more blocks.
    ///
    /// Future versions of Zebra may change the required and optional checkpoints.
```